modules/_labs/select/react/stories/stories_CypressTesting.tsx

```diff
@@ -1,167 +1,20 @@
 /// <reference path="../../../../../typings.d.ts" />
 import * as React from 'react';
-import {StaticStates} from '@workday/canvas-kit-labs-react-core';
-import {
-  ComponentStatesTable,
-  permutateProps,
-  withSnapshotsEnabled,
-  controlComponent,
-  customColorTheme,
-} from '../../../../../utils/storybook';
-import Select from '../lib/Select';
-import SelectBase from '../lib/SelectBase';
-import SelectOption from '../lib/SelectOption';
+import {controlComponent} from '../../../../../utils/storybook';
 
 import {colors} from '@workday/canvas-kit-react-core';
 import {Button} from '../../../../button/react';
 import FormField from '../../../../form-field/react';
 import {Modal, useModal} from '../../../../modal/react';
 
-import {manyOptions, options} from './stories';
+import Select from '../lib/Select';
 
-const normalizedOptions = options.map(option => {
-  return {
-    data: {},
-    disabled: option.disabled || false,
-    id: option.value,
-    label: option.label || option.value,
-    value: option.value,
-  };
-});
+import {manyOptions, options} from './stories';
 
-export default withSnapshotsEnabled({
-  title: 'Testing/React/Labs/Select',
+export default {
+  title: 'Testing/React/Labs/Select/Cypress',
```

Comment:
There were a number of stories in `stories_VisualTesting.tsx` that don't require snapshots and never should have been there in the first place (stories used for Cypress and manual testing). I've moved them to `stories_CypressTesting.tsx.` Following [the convention](https://workday.github.io/canvas-kit/?path=/story/testing-react-labs-side-panel--side-panel-states) established by the Labs Side Panel, these Cypress stories are contained in a separate `Cypress` folder in Storybook (one level down from the visual testing stories).